### ISSUE TITLE
Waveforms passed to network have incorrect magnitudes 

### ISSUE THREAD

**Issue by EthanMarx** [ML4GW/BBHNet#80]
Looking at examples of signals right before entering the network (pre-whitening) have really wrong magnitudes. I checked the raw signal files, and the magnitudes there look okay. Thus, something must be going wrong in the projection step. See example below, where magnitudes are of order `1e-3`.
<img width="408" alt="Screen Shot 2022-06-17 at 11 30 08 AM" src="https://user-images.githubusercontent.com/61295922/174330243-e1c31686-0d50-4090-acc1-a23aa03518e9.png">

**Comment by EthanMarx** [comment]
This may be directly related to #74

**Comment by alecgunny** [comment]
So in the example above, this is a plot of the H and L signals after injection but pre-whitening, but the reason they look like they're basically just pure waveforms is that their amplitude is ~19 orders of magnitude higher than the background amplitude?

**Comment by EthanMarx** [comment]
Exactly. I resolved this by removing the `np.clip` function. I think we should put a check somewhere in the `dataloader`, or maybe in the background data generation project, that makes sure the `asds` look okay (i.e. no `nans`, or 0s)
